index.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header class="beginning">
+        <nav class="head">
+            <div class="bruw blue">
+                <h2>My Logo</h2>
+            </div>
+            <div class="menu"><img src="images/Hamburger button.svg"></div>
+            <div class="navigation_option">
+                <a class="">Portfolio</a>
+                <a class="">About</a>
+                <a class="">Contact</a>
+            </div>
+        </nav>
+    </header>
+    <main id="toppart">
+        <section class="toolbar">
+            <div class="middle">
+                <h1 class="start">I'm Lethabo Glad to <br>see you!</h1>
+                <div>
+                    <p class="mid"> I’m a software developer! I can help you build a product , feature or website Look <br>through some of my work and experience! If you like what you see<br> and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+
+                <p><a class=" blu">LET'S CONNECT</a></p>
+                <ul class="socials">
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/Twitter.svg" alt="Twitter" /></a>
+                    </li>
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/Linkedin.svg" alt="Linkedin" /></a>
+                    </li>
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/Medium.svg" alt="Meduim" /></a>
+                    </li>
+                    <li class="link">
+                        <a href=""><img class="astaricks" src="images/github.png" alt="github" /></a>

```

Comment:
The name of this icon is `GitHub.svg`. Because the name is not spelt right, the icon is not showing.
![image](https://user-images.githubusercontent.com/83596135/203445663-ad24b624-044f-449a-88b5-2d1a8d6fa4fb.png)

- [ ] Please correct the path with the correct name in order for this icon to be displayed. :+1: